src/api/github/__mocks__/octokitWithRetries.ts

```diff
@@ -0,0 +1,181 @@
+import MockCompareCommits from '@test/compareCommits.json';
+import { workflow_run_job } from '@test/payloads/github/workflow_run_job';
+
+import { MockOctokitError } from './mockError';
+
+// Looks goofy, don't it? This is what I could figure out to mock a class. I
```

Comment:
What were you aiming for here? I think a manual mock here seems fine as the implementation details are clear and can be used across files